curvefs/src/client/rpcclient/mds_client.h

```diff
@@ -162,6 +162,10 @@ class MdsClientImpl : public MdsClient {
  private:
     FSStatusCode ReturnError(int retcode);
 
+    template<typename T>
+    void GetEndPoint(const T &info, butil::EndPoint *internal,
+        butil::EndPoint *external);
```

Comment:
no need to define it as member functions, define in cpp file and as free template function is enough.

Reply:
> no need to define it as member functions, define in cpp file and as free template function is enough.

fixed